welcome/how-ias-work.md

```diff
@@ -25,7 +25,7 @@ File | Purpose | Location
 [`GreatestCommonFactor.pm`](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/GreatestCommonFactor.pm) | A Perl file that specifies the query triggers and calculate the answer. | Perl files are placed in the [`zeroclickinfo-goodies/lib/DDG/Goodie`](https://github.com/duckduckgo/zeroclickinfo-goodies/tree/master/lib/DDG/Goodie) directory.
 [`GreatestCommonFactor.t`](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/t/GreatestCommonFactor.t) | A test file; it asserts that specific search queries will trigger (or not trigger) this Instant Answer, and what responses to expect | Test files are placed in the [`zeroclickinfo-goodies/t`](https://github.com/duckduckgo/zeroclickinfo-goodies/tree/master/t) directory.
 
-There is no need for front end display files because this Goodie uses the simplest form of displaying results: [structured responses](http://docs.duckduckhack.com/frontend-reference/setting-goodie-display.html#easy-structured-responses).
+There is no need for frontend files because the display is simple enough to [set our frontend properties in the Perl](http://docs.duckduckhack.com/frontend-reference/setting-goodie-display.html).
```

Comment:
My gut feeling is to have a higher threshold for using a template than two usages. My thinking is that in contrast to refactoring code or applying DRY, there's value to keeping text in line for editing as much as possible to easily understand what the reader will see without needing to reference another document.